src/resmom/linux/mom_mach.c

```diff
@@ -8001,6 +8008,66 @@ pidcache_create(void)
 	return (pidcache_arena);
 }
 
+/** @fn pidcache_reset
+ * @brief	reset pidcache with new set of PIDs from /dev/cpuset/PBSPro
+ *
+ * @return	int
+ * @retval	nprocs	- number of processes cached
+ *		-1	- when stat fails to open a cpuset tasks file
+ *
+ * @par MT-Safe:	yes
+ * @par Side Effects:
+ *	None
+ */
+static int
+pidcache_reset(pidcachetype_t *pidcache)
+{
+	unsigned long long	numerator, denominator;
+	unsigned long		setsize;
+	pidcachetype_t		*new_set;
```

Comment:
Looks like some unused variables are being declared that will likely result in compiler warnings. Please ensure your build log is free of warnings. There is only one expected warning related to ical.h.